meraki_appliance.tf

```diff
@@ -3,21 +3,25 @@ locals {
     for domain in try(local.meraki.domains, []) : [
       for organization in try(domain.organizations, []) : [
         for network in try(organization.networks, []) : {
-          network_id = meraki_network.network["${organization.name}/${network.name}"].id
-          data       = try(network.appliance.content_filtering, null)
+          key                    = format("%s/%s/%s", domain.name, organization.name, network.name)
```

Comment:
Why is this changed to `format()` instead of string interpolation?

I recognize that both are in use, which is inconsistent, but why not use string interpolation everywhere instead? It looks more readable / grep'able to me.

Reply:
Both are fine. This looks slightly more readable to me, but just a personal preference.